src/App.tsx

```diff
@@ -80,7 +79,7 @@ const App = (): JSX.Element => {
 
   return (
     <Layout>
-      {process.env.REACT_APP_BITCOIN_NETWORK === BitcoinNetwork.Testnet && <TestnetBanner />}
```

Comment:
We can remove the legacy testnet banner and use the new SiteInformation component instead.